QATCH/ui/runInfo.py

```
@@ -2377,17 +2460,21 @@ def show_hide_gui(self, object):
                 if item.widget():
                     item.widget().deleteLater()
 
-            self.groupSolvent.setVisible(is_bioformulation == False)  # solvent group
-            self.groupProtein.setVisible(is_bioformulation == True)  # protein group
-            self.groupBuffer.setVisible(is_bioformulation == True)  # buffer group
+            self.groupSolvent.setVisible(
+                is_bioformulation == False)  # solvent group
```

Ruff (0.15.9)

[error] Avoid equality comparisons to `False`; use `not is_bioformulation:` for false checks

Replace with `not is_bioformulation`

(E712)